hexrd/fitting/fitpeak.py

```diff
@@ -198,13 +195,13 @@ def fit_pk_parms_1d(p0, x, f, pktype='pvoigt'):
     weight = np.max(f)*10.  # hard coded should be changed
     fitArgs = (x, f, pktype)
     if pktype == 'gaussian':
-        p, outflag = optimize.leastsq(
+        p, _outflag = optimize.leastsq(
```

Comment:
Is `_outflag` even used anywhere? If not, consider replacing all instances of it with `_`.